templates/security-series/index.html

```diff
@@ -57,30 +59,30 @@ <h1>Ubuntu Security:<br />Security and compliance for the full stack</h1>
 
   <div class="row u-equal-height">
     <div class="col-6" >
-      <p class ="p-heading--4">Team</p>
+      <p class ="p-heading--4"><a href="#team">Team based webinars</a></p>
       <hr />
-      <ul class="p-list">
-        <li class="p-list__item"><a href="#management-series">CISO|CTO &nbsp;&rsaquo;</a></li>
-        <li class="p-list__item"><a href="#engineering-series">DevSecOps &nbsp;&rsaquo;</a></li>
-        <li class="p-list__item"><a href="#management-series">IT Security Manager, Director &nbsp;&rsaquo;</a></li>
-        <li class="p-list__item"><a href="#engineering-series">Security, Systems Engineer &nbsp;&rsaquo;</a></li>
-        <li class="p-list__item u-sv4"><a href="#management-series">VP Engineering, Ops &nbsp;&rsaquo;</a></li>
+      <ul class="p-inline-list--middot">
+        <li class="p-inline-list__item"><a href="#management-series">CISO|CTO</a></li>
+        <li class="p-inline-list__item"><a href="#engineering-series">DevSecOps</a></li>
+        <li class="p-inline-list__item"><a href="#management-series">IT Security Manager, Director</a></li>
+        <li class="p-inline-list__item"><a href="#engineering-series">Security, Systems Engineer</a></li>
+        <li class="p-inline-list__item u-sv4"><a href="#management-series">VP Engineering, Ops</a></li>
       </ul>
     </div>
     <div class="col-6">
-      <p class ="p-heading--4">Topic</p>
+      <p class ="p-heading--4"><a href="#topic">Topic based webinars</a></p>
       <hr />
-      <ul class="p-list">
-        <li class="p-list__item"><a href="#iot-security">Compliance tech &nbsp;&rsaquo;</a></li>
-        <li class="p-list__item"><a href="#iot-security">IoT & Device security &nbsp;&rsaquo;</a></li>
-        <li class="p-list__item"><a href="#multi-cloud-security">Multi-cloud security &nbsp;&rsaquo;</a></li>
-        <li class="p-list__item"><a href="#securing-public-cloud">Securing public cloud &nbsp;&rsaquo;</a></li>
-        <li class="p-list__item"><a href="#security-patching">Security Patching &nbsp;&rsaquo;</a></li>
+      <ul class="p-inline-list--middot">
+        <li class="p-inline-list__item"><a href="#iot-security">Compliance tech</a></li>
+        <li class="p-inline-list__item"><a href="#iot-security">IoT & Device security</a></li>
+        <li class="p-inline-list__item"><a href="#multi-cloud-security">Multi-cloud security</a></li>
+        <li class="p-inline-list__item"><a href="#securing-public-cloud">Securing public cloud</a></li>
+        <li class="p-inline-list__item"><a href="#security-patching">Security Patching</a></li>
       </ul>
     </div>
   </div>
 </section>
-<section class="p-strip--light is-deep" id="engineering-series">
+<section id="team" class="p-strip--light is-deep" id="engineering-series">
```

Comment:
@pmahnke  This <section> has two IDs - so the nav links pointing to 'engineering-series' don't redirect to this section, only the 'Team based webinars' link does. Maybe you could remove the id="team" and point the 'Team based webinars' to 'engineering-series'?

Reply:
@bethcollins92  good catch... I did that for topic as well... I have pushed up a fix.

Reply:
LGTM!